### ISSUE TITLE
Some improvements

### ISSUE THREAD

**Issue by Spontini** [AnvilPowered/OnTime#2]
1-) There should be a way to set some custom commands that'll be executed by the console when a person ranks up. 
2-) There should be a way to change the `/ot` into `/ru` because that's what 90% of the player base used to use.
3-) Playtime shouldn't be the only wanted thing for rankingup (most likely there should be an option), some good ways to do it may be some stuff like this;

```
Default > Member
Requirements:
100 Minutes Gameplay
One or more specificted by the config Advancement. (For example you need to get X and Y advancement due rankup)
50 Mob Kills
```

and if a person completes all of the requirements they rankup.

**Comment by alexstaeding** [comment]
1) This is something I'm working on.
2) I don't think adding an alias from another plugin is a good idea. Its just two letters and you get used to it after the first time you write it.
3) I may do this, I would need to find a good way of writing this in the config. Additionally, I would need to track this data in a database somewhere and I don't like the idea of tracking kills in a time tracker plugin.

Currently, the config looks like this, which I like because of the simplicity.
```
# Player ranks and their time requirement in seconds.
ranks {
    noob=0
    player=600
    trusted=1800
}
```
The issue I have with adding a bunch of these extra requirements is that it adds a bunch of extra lines to the config that most people won't use. Having one line per rank is very simple and is enough for most people.

When it comes to commands, I'll probably end up adding a separate map like this:
```
# Commands to run after a player has received a rank. The nodes are compiled as a regex
# and compared with the new rank name.
# Available placeholders:
# - %player%
# - %rank%
# - %time%
commands {
    # Commands that run for every rank
    ".*"=[
        "say %player% has advanced to %rank% after playing for %time%",
        "give %player% diamond 1"
    ]
    # Commands that run only for "noob" (in addition to the above)
    "noob"=[
        "say %player% is a noob"
    ]
}

**Comment by Spontini** [comment]
What about this, for normal users you can just the simplistic config and for advanced users you can have the advanced config in your WIKI, which we can basically replace easily and do edits on it.

**Comment by haxagon1990** [comment]
I love how simple this plugin is but.... is this just making its own rank system? if so we need a way to attach commands to each rank, then this could easily be one of the most useful plugins.

**Comment by LGCMcLovin** [comment]
Haxagon1990, if you are referring to permissions  to commands based on ranks, this plugin can do this

**Comment by alexstaeding** [comment]
Implemented in 5224210575641f8311d597044e3892959f3a4028
